docs/blog/microbit/2020-release.md

```diff
@@ -0,0 +1,81 @@
+# MakeCode for the micro:bit – 2020 Beta Release 
+
+It’s springtime, and even though all of us are working from home these days, the release cycle doesn’t stop! We are happy to announce the Beta for our 2020 MakeCode for the micro:bit release which is still scheduled to ship for general availability in June.
+
+Thank you to everyone who has helped us identify bugs and suggested new features throughout the year. And special Thank You to our translators and proofreaders who have helped us localize in 28 different languages! Remember, we have support for in-context translations (https://makecode.com/blog/translations-in-context), so it’s easier than ever to localize MakeCode for your language.
+
+This year’s update has a lot of juicy new features and improvements, which are listed below. Please try out the Beta and let us know what you think!
+
+https://makecode.microbit.org/beta Any bugs you find, please log them in GitHub: https://github.com/Microsoft/pxt-microbit/issues
+
+Any other comments, suggestions, and feedback, please participate in the micro:bit community on Slack: https://tech.microbit.org/get-involved/where-to-find/
+
+## New features for this release
+
+Here’s a run-down of the new features in the 2020 release:
+
+### Functions with return values
+
+Can’t get enough of Functions! Last year we added functions with parameters. You loved it and asked for more. So, this year, we’re adding Functions with return values. Here’s a little explanation of functions – 
+https://youtu.be/1LACtv9XvXQ
+
+### Expand/Collapse Blocks
+
+We know that some power-users out there have been writing very large Block programs which can be hard to navigate. We’ve added the ability to collapse and expand clusters of blocks and Functions to help tidy the Workspace and make large programs more readable. 
+
+![Expanding and collapsing blocks](/static/blog/microbit/2020-release/expand-collapse.gif)
+
+### Python language support
+
+Some of you may know that we’ve had support for the Python programming language as well as JavaScript in MakeCode for Minecraft for a few months now (https://makecode.com/blog/minecraft/python-release). We’ve gotten a lot of positive feedback from Educators about this, so we’re excited to bring it to micro:bit in this release!
+
+https://youtu.be/G3bW_AMyDHA
+
+### Debugger
+
+Many Computing Education Researchers define knowledge of a programming language to be the mapping between syntax, semantics, and state. I think all too often we focus on syntax and semantics in our educational tools. No more! Debuggers are critical tools to help students understand the state of their program execution flow. We’ve worked hard to add debug capabilities to both Block and Text editors. 
```

Comment:
The debugger replaces and extends the slow-mode.

Reply:
Slo-Mo is still there. Will it get chopped in the release?

![image](https://user-images.githubusercontent.com/27789908/80507586-d64a1600-892b-11ea-9bce-a74d3ee8a5e5.png)

Reply:
Yes, plan is to remove the Slo-Mo button, but it hasn't happened yet